src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -21,6 +21,8 @@ const typesValuesList = typesList.reduce(
   (acc, { key, value, info }) => ({ ...acc, [key]: { value, info } }),
   {}
 )
+const filterSources = (sources) =>
```

Comment:
No need to use `=== true`:

```javascript
const filterSources = (sources) =>   sources.filter(({ can_be_selected }) => can_be_selected)
```

Personally, I would name the function `selectableSources`.

Reply:
both done👍